core/src/main/scala/sttp/openai/OpenAi.scala

```diff
@@ -86,10 +87,14 @@ class OpenAi(authToken: String) {
   /** Creates edited or extended images given an original image and a prompt
     *
     * @param systemPath
+    *   <<<<<<< HEAD SystemPath of the JSON Lines image to be edited. <p> Must be a valid PNG file, less than 4MB, and square. If mask is
+    *   not provided, image must have transparency, which will be used as the mask
+    * \=======
     *
     * SystemPath of the JSON Lines image to be edited. <p> Must be a valid PNG file, less than 4MB, and square. If mask is not provided,
     * image must have transparency, which will be used as the mask
     *
+    * >>>>>>> 7ddcc2c5fbc796c3c585c545ce5aadfccd0e949b
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**nitpick:** ‏remove git's diff comments like `<<<<<<< HEAD` etc.